src/backendbot/api_routes.py

```
@@ -678,3 +539,399 @@ def get_logs(limit: int = 100) -> list[dict[str, Any]]:
+        return {"event_logs": result.stdout}
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=f"Error obteniendo logs de eventos: {e}")
+
+
+@router.post("/schedule-task", dependencies=[Depends(get_api_key)])
+def schedule_task(task_name: str, command: str, schedule: str):
+    """Programa una tarea en el programador de tareas."""
+    try:
+        import subprocess
+        # Crear tarea básica (ejemplo simple)
+        subprocess.run(['schtasks', '/create', '/tn', task_name, '/tr', command, '/sc', schedule], capture_output=True)
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)